### ISSUE TITLE
[QUESTION] Where I can see next version (bug fixes, not minor/major) release timeline?

### ISSUE THREAD

**Issue by kfirpeled** [litedb-org/LiteDB#1671]
There's a bug fix I'm waiting for, and I was wondering if there is an estimated timetable for next bug fixes version

Of course I'll be more than happy to assist in anyway to push the next release sooner :)

**Comment by maxkatz6** [comment]
Agree. In past LiteDB's developers said, that bugfixes arrives with next planned release. And LiteDB doesn't have nightly builds.
So, if every release is planned, shouldn't information about date be available for library users?
Thanks.

**Comment by JKamsker** [comment]
There's no published calendar. Since this was asked, though, the release process has changed, and fixes no longer wait for a planned release. Every merge to `dev` that touches the library is built and, once CI passes, pushed to NuGet automatically as a prerelease (`6.0.0-prerelease.N`) by [`publish-prerelease.yml`](https://github.com/litedb-org/LiteDB/blob/dev/.github/workflows/publish-prerelease.yml). Once a fix is merged, you can pick it up right away:

```
dotnet add package LiteDB --prerelease
```

Stable releases (latest: 5.0.21) are still cut manually. #2623 tracks the v6 planning.

Closing this as answered — if it doesn't cover your case, comment here and we'll reopen it.
